apps/docs/src/routes/+layout.svelte

```diff
@@ -9,12 +9,9 @@
 	import '$lib/theme/theme.css';
 	import type { PageData } from './$types';
 
-	let window_width = 0;
-	let show_sidebar: boolean;
+	let show_sidebar: boolean = false;
 
-	$: mobile = window_width < 800;
 	$: nosidebar = browser && $page.url.pathname === '/';
```

Comment:
I think removing the `browser` should fix the pop in of the sidebar on first reload